src/main/kotlin/mate/academy/model/Circle.kt

```diff
@@ -0,0 +1,15 @@
+package mate.academy.model
+
+class Circle(color: Color, private val radius: Double) : Figure (color) {
+
+    override fun getArea(): Double = Math.PI * radius * radius
+
+    override fun draw(): String = ("Figure: ${javaClass.simpleName.lowercase()}, "
+            + "area: ${getArea()} sq. units, "
+            + "radius: $radius units, "
+            + "color: ${color.toString().lowercase()}"
+//            + javaClass.getDeclaredFields().asSequence()
+//                .map { "${it.name}: ${this.get(it.name)}"}
+//                .joinToString(", ")
```

Comment:
fix imports and use text block instead of concatenation

Reply:
Concatenation is using instead of text block, because output is one-line, line of code is very long, and for more readability (1 parameter - in 1 line of code)